packages/iov-keycontrol/src/wallets/ed25519wallet.spec.ts

```diff
@@ -211,6 +211,43 @@ describe("Ed25519Wallet", () => {
     );
   });
 
+  it("can export the secret in a printable format", async () => {
+    {
+      const wallet = new Ed25519Wallet();
+      await wallet.createIdentity(
+        Ed25519Keypair.fromLibsodiumPrivkey(
+          fromHex(
+            "0000000000000000aaaaaaaaaaaaaaaa1111111111111111dddddddddddddddd7777777777777777bbbbbbbbbbbbbbbb5555555555555555ffffffffffffffff",
+          ),
+        ),
+      );
+      expect(wallet.printableSecret()).toEqual(
+        "0000000000000000 aaaaaaaaaaaaaaaa 1111111111111111 dddddddddddddddd 7777777777777777 bbbbbbbbbbbbbbbb 5555555555555555 ffffffffffffffff",
```

Comment:
spaces for readability?

Reply:
Yeah for giving humans a chance to restore it from a paper backup. Still no fun but possible